might be a fluke since it was in a test 
```
Successfully built langchain_core-0.2.1-py3-none-any.whl
Finished creating a wheel...
/build/source/libs/core/dist /build/source/libs/core
Unpacking to: unpacked/langchain_core-0.2.1...OK
Repacking wheel as ./langchain_core-0.2.1-py3-none-any.whl...creating './langchain_core-0.2.1-py3-none-any.whl' and adding 'un>
adding 'langchain_core/__init__.py'
adding 'langchain_core/agents.py'
adding 'langchain_core/caches.py'
adding 'langchain_core/chat_history.py'
adding 'langchain_core/chat_loaders.py'
adding 'langchain_core/chat_sessions.py'
adding 'langchain_core/env.py'
adding 'langchain_core/exceptions.py'
adding 'langchain_core/globals.py'
adding 'langchain_core/memory.py'
adding 'langchain_core/prompt_values.py'
adding 'langchain_core/py.typed'
adding 'langchain_core/retrievers.py'
adding 'langchain_core/stores.py'
adding 'langchain_core/structured_query.py'
adding 'langchain_core/sys_info.py'
adding 'langchain_core/tools.py'
adding 'langchain_core/vectorstores.py'
adding 'langchain_core/_api/__init__.py'
adding 'langchain_core/_api/beta_decorator.py'
adding 'langchain_core/_api/deprecation.py'
adding 'langchain_core/_api/internal.py'
adding 'langchain_core/_api/path.py'
adding 'langchain_core/beta/__init__.py'
adding 'langchain_core/beta/runnables/__init__.py'
adding 'langchain_core/beta/runnables/context.py'
adding 'langchain_core/callbacks/__init__.py'
adding 'langchain_core/callbacks/base.py'
adding 'langchain_core/callbacks/file.py'
...skipping...
            "passthrough": RunnablePassthrough(),
        }
    
        stream = chain.stream({"question": "What is your name?"})
    
        final_value = None
        streamed_chunks = []
        for chunk in stream:
            streamed_chunks.append(chunk)
            if final_value is None:
                final_value = chunk
            else:
                final_value += chunk
    
>       assert streamed_chunks[0] in [
            {"passthrough": prompt.invoke({"question": "What is your name?"})},
            {"llm": "i"},
            {"chat": AIMessageChunk(content="i")},
        ]
E       AssertionError: assert {'chat': AIMessageChunk(content='i', id='run-ced90791-e3fa-4398-8ab7-35b3d95acb24')} in [{'pass>

tests/unit_tests/runnables/test_runnable.py:3263: AssertionError
=============================== warnings summary ===============================
tests/unit_tests/prompts/test_few_shot.py:25
tests/unit_tests/prompts/test_few_shot.py:25
tests/unit_tests/prompts/test_few_shot.py:25
tests/unit_tests/prompts/test_few_shot.py:25
tests/unit_tests/prompts/test_few_shot.py:25
tests/unit_tests/prompts/test_few_shot.py:25
tests/unit_tests/prompts/test_few_shot.py:25
tests/unit_tests/prompts/test_few_shot.py:25
  /build/source/libs/core/tests/unit_tests/prompts/test_few_shot.py:25: PytestRemovedIn9Warning: Marks applied to fixtures hav>
  See docs: https://docs.pytest.org/en/stable/deprecations.html#applying-a-mark-to-a-fixture-function
  ```

--- a/pkgs/by-name/op/open-webui/package.nix
+++ b/pkgs/by-name/op/open-webui/package.nix
@@ -7,19 +7,19 @@
 }:
 let
   pname = "open-webui";
-  version = "0.2.5";
+  version = "0.3.2";
 
   src = fetchFromGitHub {
     owner = "open-webui";
     repo = "open-webui";
     rev = "v${version}";
-    hash = "sha256-BqclBrkFmTUJZuUySyNsQFlfdyENaIhqjn50797zJHY=";
+    hash = "sha256-hUm4UUQUFoDRrAg+RqIo735iQs8304OUJlT91vILmXo=";
   };
 
   frontend = buildNpmPackage {
     inherit pname version src;
 
-    npmDepsHash = "sha256-drMUcCUeyPEnaa1fSsfHJfInenyfH7NNJgBFeRsfOuE=";
+    npmDepsHash = "sha256-VdGneemYLMuMczjQB6I35Ry2kyIuAe2IaeDus/NvzK8=";
 
     # Disabling `pyodide:fetch` as it downloads packages during `buildPhase`
     # Until this is solved, running python packages from the browser will not work.
@@ -55,64 +55,63 @@ python3.pkgs.buildPythonApplication rec {
   pythonRelaxDeps = true;
 
   pythonRemoveDeps = [
-    # using `psycopg2` instead
-    "psycopg2-binary"
     # using `opencv4`
     "opencv-python-headless"
+    # using `psycopg2` instead
+    "psycopg2-binary"
     # package request: https://github.com/NixOS/nixpkgs/issues/317065
     "rapidocr-onnxruntime"
-    # package request: https://github.com/NixOS/nixpkgs/issues/317066
-    "langfuse"
-    # package request: https://github.com/NixOS/nixpkgs/issues/317068
-    "langchain-chroma"
   ];
 
   dependencies = with python3.pkgs; [
+    aiohttp
+    apscheduler
+    argon2-cffi
+    bcrypt
     beautifulsoup4
+    black
+    boto3
+    chromadb
+    docx2txt
+    extract-msg
+    fake-useragent
     fastapi
-    uvicorn
-    python-multipart
+    faster-whisper
     flask
     flask-cors
-    python-socketio
-    python-jose
+    fpdf2
+    google-generativeai
+    langchain
+    langchain-chroma
+    langchain-community
+    langfuse
+    litellm
+    markdown
+    opencv4
+    openpyxl
+    pandas
     passlib
-    requests
-    aiohttp
     peewee
     peewee-migrate
     psycopg2
-    pymysql
-    bcrypt
-    litellm
-    boto3
-    argon2-cffi
-    apscheduler
-    google-generativeai
-    langchain
-    langchain-community
-    fake-useragent
-    chromadb
-    sentence-transformers
-    pypdf
-    docx2txt
-    python-pptx
-    unstructured
-    markdown
-    pypandoc
-    pandas
-    openpyxl
-    pyxlsb
-    xlrd
-    validators
-    opencv4
-    fpdf2
-    rank-bm25
-    faster-whisper
     pyjwt
-    black
-    youtube-transcript-api
+    pymysql
+    pypandoc
+    pypdf
+    python-jose
+    python-multipart
+    python-pptx
+    python-socketio
     pytube
+    pyxlsb
+    rank-bm25
+    requests
+    sentence-transformers
+    unstructured
+    uvicorn
+    validators
+    xlrd
+    youtube-transcript-api
   ];
 
   build-system = with python3.pkgs; [

